packages/api-plugin-accounts/src/queries/filterSearchAccounts.js

```diff
@@ -0,0 +1,24 @@
+import generateFilterQuery from "@reactioncommerce/api-utils/generateFilterQuery.js";
+
+/**
+ * @name filterSearchAccounts
+ * @method
+ * @memberof GraphQL/Accounts
+ * @summary Query the Accounts collection for a list of customers/accounts
+ * @param {Object} context - an object containing the per-request state
+ * @param {Object} conditions - object containing the filter conditions
+ * @param {String} shopId - shopID to filter by
+ * @returns {Promise<Object>} Accounts object Promise
+ */
+export default async function filterSearchAccounts(context, conditions, shopId) {
+  const { collections: { Accounts } } = context;
+
+  if (!shopId) {
+    throw new Error("shopId is required");
+  }
+  await context.validatePermissions("reaction:legacy:accounts", "read", { shopId });
+
+  const { filterQuery } = generateFilterQuery(context, "Account", conditions, shopId);
```

Comment:
The fact that this is `Account` and not `Accounts` seems worrisome

Reply:
It is 'Account' in SimpleSchema.
```
  Account: SimpleSchema {
    pick: [Function: pickOrOmit],
    omit: [Function: pickOrOmit],
    _constructorOptions: { humanizeAutoLabels: true, requiredByDefault: true },
    _cleanOptions: {
      autoConvert: true,
      extendAutoValueContext: {},
      ...
```